packages/addons/service/tvheadend42/source/defaults/epggrab/config

```diff
@@ -5,6 +5,69 @@
 	"epgdb_periodicsave": 2,
 	"ota_initial": true,
 	"modules": {
+		"opentv-skyit": {
+			"class": "epggrab_mod_ota",
+			"name": "OpenTV: Sky Italia",
+			"type": "Over-the-air",
+			"enabled": true,
+			"priority": 2
+		},
+		"opentv-skynz": {
+			"class": "epggrab_mod_ota",
+			"name": "OpenTV: Sky NZ",
+			"type": "Over-the-air",
+			"enabled": true,
+			"priority": 2
+		},
+		"opentv-skyuk": {
+			"class": "epggrab_mod_ota",
+			"name": "OpenTV: Sky UK",
+			"type": "Over-the-air",
+			"enabled": true,
+			"priority": 2
+		},
+		"opentv-ausat": {
+			"class": "epggrab_mod_ota",
+			"name": "OpenTV: Ausat",
+			"type": "Over-the-air",
+			"enabled": true,
+			"priority": 2
+		},
+		"psip": {
+			"class": "epggrab_mod_ota",
+			"name": "PSIP: ATSC Grabber",
+			"type": "Over-the-air",
+			"enabled": true,
+			"priority": 1
+		},
+		"Bulsatcom_39E": {
+			"class": "epggrab_mod_ota",
+			"name": "Bulsatcom: Bula 39E",
+			"type": "Over-the-air",
+			"enabled": true,
+			"priority": 5
+		},
+		"viasat_baltic": {
+			"class": "epggrab_mod_ota",
+			"name": "VIASAT: Baltic",
+			"type": "Over-the-air",
+			"enabled": true,
+			"priority": 5
+		},
+		"uk_freeview": {
+			"class": "epggrab_mod_ota",
+			"name": "UK: Freeview",
+			"type": "Over-the-air",
+			"enabled": true,
+			"priority": 5
+		},
+		"uk_freesat": {
+			"class": "epggrab_mod_ota",
+			"name": "UK: Freesat",
+			"type": "Over-the-air",
+			"enabled": true,
+			"priority": 5
+		},
```

Comment:
I'ts the default by tvh, so that should be save (and I'm using that for years):
https://github.com/tvheadend/tvheadend/blob/master/src/epggrab.c#L175

We could also remove the config and use tvheadends defaults? We will lose the periodic save and channel rename. 

Btw: on 4.0, periodic save causes glitches in the livestreams/recordings when saving. Don't know if that's fixed.